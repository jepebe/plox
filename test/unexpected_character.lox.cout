[38;5;1m[line 3] Error: Unexpected character: '|'[0m
1 error(s) and 0 warning(s) occurred
