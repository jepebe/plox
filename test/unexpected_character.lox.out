[38;5;1m[line 3] Error: Unexpected character: '|'[0m
[38;5;1m[line 3] Error at 'b': Expect ')' after arguments.[0m
2 error(s) and 0 warning(s) occurred
